### PR TITLE
[DOC] Update doc for HBase interpreter

### PR DESCRIPTION
**Pull request by felixcheung**
### What is this PR for?
Update HBase interpreter for style and additional info

### What type of PR is it?
Documentation

### Todos
* [x] - Update doc

### Is there a relevant Jira issue?
N/A

### How should this be tested?
N/A

### Screenshots (if appropriate)
N/A

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? This is

### CODE REVIEW

**Comment by felixcheung** [comment]
@vrajat @AhyoungRyu please review.

File: docs/interpreter/hbase.md [hunk 1]
```diff
@@ -6,15 +6,13 @@ group: manual
...
-
 ## HBase Shell Interpreter for Apache Zeppelin
-[HBase Shell](http://hbase.apache.org/book.html#shell) is a JRuby IRB client for Apache HBase. 
-This interpreter provides all capabilities of Apache HBase shell within Apache Zeppelin. The 
-interpreter assumes that Apache HBase client software has been installed and its possible to 
-connect to the Apache HBase cluster from the machine on where Apache Zeppelin is installed.
+[HBase Shell](http://hbase.apache.org/book.html#shell) is a JRuby IRB client for Apache HBase. This interpreter provides all capabilities of Apache HBase shell within Apache Zeppelin. The interpreter assumes that Apache HBase client software has been installed and its possible to connect to the Apache HBase cluster from the machine on where Apache Zeppelin is installed.  
+To start with HBase, please see [HBase Quickstart](https://hbase.apache.org/book.html#quickstart)
+
+> Note: currently only HBase 1.0.x releases are supported.
```
Comment (felixcheung) [inline docs/interpreter/hbase.md]: I tested with HBase 1.1.3 and it didn't work.

File: docs/interpreter/hbase.md [hunk 2]
```diff
@@ -39,19 +37,31 @@ connect to the Apache HBase cluster from the machine on where Apache Zeppelin is
...
 ```bash
-| %hbase
-| help
+%hbase
+put 'test', 'row1', 'cf:a', 'value1'
 ```
 
-For more information on all commands available, refer to [HBase Shell Documentation](http://hbase.apache.org/book.html#shell)
\ No newline at end of file
+For more information on all commands available, refer to [HBase shell commands](https://learnhbase.wordpress.com/2013/03/02/hbase-shell-commands/)
```
Comment (felixcheung) [inline docs/interpreter/hbase.md]: This is also linked to in HBase official documentation, should be safe.

File: docs/interpreter/hbase.md [hunk 3]
```diff
@@ -6,15 +6,13 @@ group: manual
 ---
 {% include JB/setup %}
 
-
 ## HBase Shell Interpreter for Apache Zeppelin
-[HBase Shell](http://hbase.apache.org/book.html#shell) is a JRuby IRB client for Apache HBase. 
-This interpreter provides all capabilities of Apache HBase shell within Apache Zeppelin. The 
-interpreter assumes that Apache HBase client software has been installed and its possible to 
-connect to the Apache HBase cluster from the machine on where Apache Zeppelin is installed.
+[HBase Shell](http://hbase.apache.org/book.html#shell) is a JRuby IRB client for Apache HBase. This interpreter provides all capabilities of Apache HBase shell within Apache Zeppelin. The interpreter assumes that Apache HBase client software has been installed and its possible to connect to the Apache HBase cluster from the machine on where Apache Zeppelin is installed.
```
Comment (AhyoungRyu) [inline docs/interpreter/hbase.md]: @felixcheung Did you intend "it's possible" ? : )

File: docs/interpreter/hbase.md (hunk 1 above)
Comment (vrajat) [inline docs/interpreter/hbase.md]: Hi,
I have ideas on how to make it work with other versions of HBase. I'll open a JIRA and I can work supporting more versions of HBase.

**Comment by vrajat** [comment]
LGTM

**Comment by AhyoungRyu** [comment]
@felixcheung Thank you for taking care of HBase docs and your effort. Looks good!

File: docs/interpreter/hbase.md (hunk 1 above)
Comment (felixcheung) [inline docs/interpreter/hbase.md]: cool - I got `cannot load Java class org.apache.hadoop.hbase.quotas.ThrottleType`

**Comment by felixcheung** [comment]
thanks, merging if no more comments.

File: docs/interpreter/hbase.md (hunk 3 above)
Comment (felixcheung) [inline docs/interpreter/hbase.md]: thx good catch

**Comment by felixcheung** [comment]
This Travis looks strange? Has this test been failing?
```
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 7.125 sec <<< FAILURE! - in org.apache.zeppelin.spark.DepInterpreterTest
testDefault(org.apache.zeppelin.spark.DepInterpreterTest)  Time elapsed: 7.051 sec  <<< ERROR!
java.lang.NullPointerException: null
	at org.sonatype.aether.impl.internal.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:352)
	at org.apache.zeppelin.spark.dep.SparkDependencyContext.fetchArtifactWithDep(SparkDependencyContext.java:179)
	at org.apache.zeppelin.spark.dep.SparkDependencyContext.fetch(SparkDependencyContext.java:129)
	at org.apache.zeppelin.spark.DepInterpreter.interpret(DepInterpreter.java:220)
	at org.apache.zeppelin.spark.DepInterpreterTest.testDefault(DepInterpreterTest.java:96)
```

**Pull request closed without merging**